source/SIL.AppBuilder.Portal.Frontend/src/ui/components/header/display.tsx

```diff
@@ -0,0 +1,64 @@
+  </span>
+)
+
+const notificationTrigger = (
+  <span>
+    <Icon circular name='alarm' size='large' />
+  </span>
+)
+
+class Header extends Component<any,any> {
```

Comment:
any,any is not good practice :)
since we don't have any props or state right now, you can just say `extends Component {...}`

otherwise, the pattern we're going with for props and state is:

```
export interface IProps {
  // ...
}

export interface IState {
  // ...
}

export class Header extends Component<IProps, IState> {
   // ...
}

export default withRouter(Header);
```